src/api/modules/decrypt/decrypt.background.ts

```diff
@@ -27,14 +25,7 @@ const background: ModuleFunction<string | Uint8Array> = async (
   isArrayBuffer(data);
 
   // grab the user's keyfile
-  const decryptedWallet = await getActiveKeyfile().catch((e) => {
```

Comment:
The logic in these `catch` blocks used in most background scripts is now included directly inside `getActiveKeyfile`.

Reply:
👍